politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -414,54 +425,105 @@ func cmdStubUsers() error {
 	return nil
 }
 
-func cmdMigrate() error {
-	// Connect to LevelDB
+func connectLevelDB() (user.Database, error) {
 	dbDir := filepath.Join(*dataDir, network)
 	_, err := os.Stat(dbDir)
 	if err != nil {
 		if os.IsNotExist(err) {
 			err = fmt.Errorf("leveldb dir not found: %v", dbDir)
 		}
-		return err
+		return nil, err
 	}
 
-	ldb, err := localdb.New(dbDir)
+	fmt.Printf("LevelDB     : %v\n", dbDir)
+	return localdb.New(dbDir)
+}
+
+func connectCockroachDB() (user.Database, error) {
+	err := validateCockroachParams()
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("new cockroachdb: %v", err)
 	}
 
-	// Connect to CockroachDB
-	err = validateCockroachParams()
+	fmt.Printf("CockroachDB : %v %v %v %v %v %v\n", *cockroachdbhost, network,
+		*rootCert, *clientCert, *clientKey, *encryptionKey)
+	return cockroachdb.New(*cockroachdbhost, network, *rootCert,
+		*clientCert, *clientKey, *encryptionKey)
+}
+
+func connectMySQLDB() (user.Database, error) {
+	err := validateMySQLParams()
 	if err != nil {
-		return fmt.Errorf("new leveldb: %v", err)
+		return nil, err
 	}
-	cdb, err := cockroachdb.New(*host, network, *rootCert,
-		*clientCert, *clientKey, *encryptionKey)
+
+	fmt.Printf("MySQLDB : %v %v\n", *mysqlhost, network)
+	return mysqldb.New(*mysqlhost, *password, network, *encryptionKey)
+}
+
+func connectDB(typeDB string) (user.Database, error) {
+	const (
+		userDBLevel     = "leveldb"
```

Comment:
These variables are only ever being referenced once. Shouldn't they be defined inline?